x/tss/client/cli/tx.go

```diff
@@ -45,7 +45,7 @@ func getCmdKeygenStart() *cobra.Command {
 		panic("flag not set")
 	}
 
-	keyRoleStr := cmd.Flags().String("key-role", exported.MasterKey.SimpleString(), "role of the key to be generated")
+	keyRoleStr := cmd.Flags().String("key-role", "", "role of the key to be generated")
```

Comment:
Avoid defaulting to master role

Reply:
We don't use threshold keys rn, so I have kept that. I want to make a minimal change because it affects our rotation instructions etc.